### ISSUE TITLE
farmer not listed in source from which to sync

### ISSUE THREAD

**Issue by Digitalin** [cosmocode/dokuwiki-plugin-farmsync#6]
Hi,

I don't see the farmer as a source from which to sync, only animals to animals. It is true with the master and refactor branch of farmer plugin.

**Comment by micgro42** [comment]
Yes, that is by design. The farmer is not intended as a source for content or structure.

**Comment by Digitalin** [comment]
Surely, you 've got a good reason for it but I am surprised of that.  Because, I imagine different use cases where farmer as a source would be helpful.

**Comment by micgro42** [comment]
The idea is to use the farmer solely as an administrative wiki and to have other animals as source for content.

Similarly, you cannot select the farmer-wiki as a template/source when creating a new animal via the farmer-plugin

However it is relatively simple to just move the content from your farmer to a (new) animal and to use that in the future.

**Comment by Digitalin** [comment]
I see :-)
so, it is well the idea of [Base Domain](https://github.com/cosmocode/dokuwiki-plugin-farmer/issues/26#event-729042212) in farmer plugin to separate a main wiki to the farmer wiki. It has probably to be explained, at least as a recommendation for a wiki farm.

**Comment by GJRobert** [comment]
It is however embarrassing that I have been using the farmer wiki long before building the farm, and it's time to move certain namespace and pages into one of the animals.

If this usage is not included in the design, may I just copy the pages manually into the animal folder? Thanks!

**Comment by Digitalin** [comment]
Yes, create a new animal with all inherited users, copy data and media into it and clean the farmer. It is easy, obviously  after backup (personally, I always experiment on a local wiki)
As I understand the design, to transform an  existing wiki into a farm, following recommendation, I see two cases : 
-  with **one** domain name, your existing wiki e.g. mywiki.**org** become animal will be reached after at **main**.mywiki.org or whatever sub domain name you give to it. 
- having a second domain name for farming e.g. mywiki.**net** gives opportunity to keep the existing wiki address (mywiki.org)  as animals can have their own FQDN and not only sub domain of farm.
